app/javascript/packs/sdk.js

```diff
@@ -31,6 +31,13 @@ const runSDK = ({ baseUrl, websiteToken }) => {
       IFrameHelper.events.toggleBubble(state);
     },
 
+    toggleBubbleVisibility(visibility) {
+      document.querySelector('.woot--bubble-holder').style.display = visibility
```

Comment:
@fayazara Please test it in the case where the message bubble is hidden already. 

Initialize it with 

```js
window.chatwootSettings = {
  hideMessageBubble: true
}
```

Reply:
Okay